### PR TITLE
CCX-5092: Add CCX troubleshooting guide

### PR DESCRIPTION
**Pull request by ram-vibhul**

### CODE REVIEW

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 1]
```diff
@@ -2,158 +2,411 @@
...
-s9s node --list --long --cluster-id=CLUSTERID
-# Check status of replication links
-s9s replication --list --long
-## List jobs and view logs
-s9s job --list
-s9s job --job-id=NNN --log
+# Step 1: Check the datastore state and get the job-id in field `Active Job:`
+ccxctl datastore state <datastore-id>
+
+# Step 2: Check the cluster state
```
[the remark this answers is not in the document]
Comment (freddielunchbird) [reply docs/admin/Troubleshooting/Troubleshooting.md:40]: Please use "datastore" if it is checking the datastore . Cluster would mean the "k8s cluster", but i am not sure which is refered to here in "Step 2:.."

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 2]
```diff
@@ -2,158 +2,411 @@
...
+ccxctl job state <job-id>
```
[the remark this answers is not in the document]
Comment (freddielunchbird) [reply docs/admin/Troubleshooting/Troubleshooting.md:41]: # Step 2: Check the cluster state
SHould be
# Step 2: Check the job state
?

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 3]
```diff
@@ -2,158 +2,411 @@
...
-## CCX Error Report
 
-The script to generate an error report is here - https://github.com/severalnines/helm-ccx/tree/main/scripts. Please attach the error report (.tar.gz file) to every ticket created for CCX support. This tremendously helps to diagnose the issue.
+# CCX Troubleshooting Guide
 
+This guide helps you troubleshoot and resolve common issues in CCX. Follow the steps outlined below to diagnose problems and apply fixes.
+
+## CCX Services Error Report
+
+To generate an error report of CCX services and components within a Kubernetes environment, follow these steps:
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Troubleshooting/Troubleshooting.md:8]: ok addressing your comments below

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 4]
```diff
@@ -1,159 +1,252 @@
...
-s9s job --job-id=NNN --log
+# Step 3: Run the script to collect logs
+./gather-logs.sh
 ```
+Attach the generated `.tar.gz` file to the CCX support ticket.
 
-### Creating an Error Report
+## Long-Running or Stuck Datastore
+Clusters may get stuck in a locked state if a job is interrupted unexpectedly. Use the following steps to resolve hanging jobs:
+Run `ccxctl` by executing inside the stores pod. Note that `ccxctl` is available from version 1.50+.
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Troubleshooting/Troubleshooting.md]: Okay updated

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 5]
```diff
@@ -1,159 +1,252 @@
...
+For prior versions, navigate to the CC UI:
+- **Clusters** -> **Select Datastore ID** -> **Nodes** -> **Actions** -> **Toggle Readonly to disable**
+Disable readonly for the nodes that are labeled Readonly.
+
+## Disk Autoscaling Issues
+Disk autoscaling is enabled by default in CCX. It can be configured using the following setting in the Helm CCX values file:
+```yaml
+autoscaling.storage.enabled: true
+```
+Ensure that the observability stack is deployed in the same namespace as CCX.
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Troubleshooting/Troubleshooting.md]: refactored

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 6]
```diff
@@ -1,159 +1,252 @@
...
+```bash
+dig +short primary.a1b3aaec-0f58-4347-80ab-27e386af2208.ccx.net
+```
+Replace the domain with the specific record you want to check.
 
-1. Select the Cluster (CCXDB or CMONDB).
-2. Select Reports, and then Create Error Report.
-3. The error report will now be created and presented in the web UI.
+#### Verify in the External DNS Provider
+Check the DNS management console of your external DNS provider. Ensure that the records are listed as expected.
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Troubleshooting/Troubleshooting.md:241]: logs and command are in next step

File: docs/admin/Troubleshooting/Troubleshooting.md [hunk 7]
```diff
@@ -1,159 +1,252 @@
...
 ```bash
-s9s cluster --list --long | grep CLUSTER_UUID
-# take the cluster id , NNN
-s9s cluster --deploy-agents --cluster-id=NNN --log
+# Step 1: Update and Deploy ccxdeps charts in the same namespace
+helm repo update
+helm upgrade --install ccxdeps s9s/ccxdeps --debug
+
+# Step 2: Verify ccxdeps-loki service is running
+kubectl get svc -n <namespace> | grep ccxdeps-loki
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Troubleshooting/Troubleshooting.md]: refactored

**Comment by ram-vibhul** [comment]
refactored your suggestions oskar

**Pull request merged**